### ISSUE TITLE
Disabled Delete Button Missing Tooltip when in Load UI Menus

### ISSUE THREAD

**Issue by RyanCountryman** [widelands/widelands#7030]
### Describe the Bug

Within the load game and load replay menus there are inconsistencies with the presentation of the hover over tooltip of the differing buttons. When no games/replays are present within these interfaces, both the Delete button and the OK button are disabled. When hovering over the OK button the tooltip “Load this game” / “Load this replay” is presented to the user, when hovering over Delete no tooltip is displayed.

### To Reproduce

**Load Game Menu**

1. Ensure no saved games are present
2. From the Main Menu, Click Single Player
3. From the populated drop down select Load Game
4. Hover over the Delete Button

**Load Replay Menu**

1. Ensure no saved replays are present
2. From the Main Menu, Click Watch Replay
3. From the populated drop down select Load Replay
4. Hover over the Delete Button

### Expected Behavior

The Delete Button should display a tooltip when inactive, consistent with the behavior of the OK button. A tooltip such as “No game selected” or “No replay selected" depending on the menu would be appropriate.

I believe this is occurring from assignment of the empty string within the else statement located in LoadOrSaveGame::set_tooltips_of_buttons found in /src/ui/shared/load_or_save_game.cc
Please see screen shot below

### Crash Log

```text

```

### Screenshots

**Disabled Delete Button on hover**
<img width="1000" height="575" alt="Image" src="https://github.com/user-attachments/assets/e44a9221-255a-41ad-9702-006a60aa11c4" />

**Disabled OK Button on hover**
<img width="1000" height="575" alt="Image" src="https://github.com/user-attachments/assets/38490875-934a-4a2c-a3cf-fb27ed2d96a8" />

**LoadOrSaveGame::set_tooltips_of_buttons**
<img width="694" height="402" alt="Image" src="https://github.com/user-attachments/assets/e86c927e-f5e4-4504-9ae8-92d143f9b6a7" />

### Operating System

Windows 10 64-bit

### Widelands Version

v1.3.1

**Comment by bunnybot** [comment]
Mirrored on Codeberg as [#CB5389](https://codeberg.org/wl/widelands/issues/5389).

**Comment by bunnybot** [comment]
[<img align='left' src='https://codeberg.org/avatars/593dc43e2444c25db9481960e5b693ab' alt='tothxa' height='48' />](https://codeberg.org/tothxa)*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/5389#issuecomment-15079161)*
*On Sat May 16 21:31:19 CEST 2026, [**Tóth András (tothxa)**](https://codeberg.org/tothxa) wrote:*

---

I think the "OK" button tooltip should also be changed to "No ... is selected" in this case then.
